bin/setup

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+project=mynamespace
```

Comment:
TODO - take this from an ENV var or command line then default to mynamespace

Reply:
I think an envvar is a good call for a first pass.

Reply:
:+1: done
Get from env var first then default to myproject